packages/govuk-frontend/src/govuk/components/skip-link/skip-link.mjs

```diff
@@ -86,57 +80,22 @@ export class SkipLink extends GOVUKFrontendComponent {
       })
     }
 
-    this.$linkedElement = $linkedElement
-
-    this.$module.addEventListener('click', () => this.focusLinkedElement())
-  }
-
-  /**
-   * Focus the linked element
-   *
-   * Set tabindex and helper CSS class. Set listener to remove them on blur.
-   *
-   * @private
-   */
-  focusLinkedElement() {
-    if (!this.$linkedElement) {
-      return
-    }
-
-    if (!this.$linkedElement.getAttribute('tabindex')) {
-      // Set the element tabindex to -1 so it can be focused with JavaScript.
-      this.$linkedElement.setAttribute('tabindex', '-1')
-      this.$linkedElement.classList.add('govuk-skip-link-focused-element')
-
-      // Add listener for blur on the focused element (unless the listener has
-      // previously been added)
-      if (!this.linkedElementListener) {
-        this.$linkedElement.addEventListener('blur', () =>
-          this.removeFocusProperties()
-        )
-        this.linkedElementListener = true
-      }
-    }
-
-    this.$linkedElement.focus()
-  }
-
-  /**
-   * Remove the tabindex that makes the linked element focusable because the
-   * element only needs to be focusable until it has received programmatic focus
-   * and a screen reader has announced it.
-   *
-   * Remove the CSS class that removes the native focus styles.
-   *
-   * @private
-   */
-  removeFocusProperties() {
-    if (!this.$linkedElement) {
-      return
-    }
-
-    this.$linkedElement.removeAttribute('tabindex')
-    this.$linkedElement.classList.remove('govuk-skip-link-focused-element')
+    /**
+     * Focus the linked element on click
+     *
+     * Adds a helper CSS class to hide native focus styles,
+     * but removes it on blur to restore native focus styles
+     */
+    this.$module.addEventListener('click', () =>
+      setFocus($linkedElement, {
+        onFocus() {
+          this.classList.add('govuk-skip-link-focused-element')
```

Comment:
Just keeping the KB file size down, but fine by me

Reply:
I've updated it in 6ef903fb78efd41110013604e584664c0a2145f7 but I'm going to keep the `.call($element)` to bind `this` as `$element` because it's what you'd expect from an event handler that isn't an arrow function

Reply:
File size has increased from ~69.9 KiB~ 69.39 KiB → 69.44 KiB which is still a saving